readme.md

```diff
@@ -0,0 +1,192 @@
+your editor to inspect and change markdown files.
+This extension is built around remark, which is a very popular ecosystem of
+plugins that work with markdown.
+You can choose from the 150+ plugins that already exist or make your own.
+
+See [remark][] for info on what the remark ecosystem is.
+
+## When should I use this?
+
+You can use this extension if you want to check (lint) and format markdown
+files from within your editor.
+
+To configure this extension, you define your preferred markdown style in a
+configuration file (`.remarkrc`, `.remarkrc.js`, etc. or in `package.json`).
+This file is picked up by `vscode-remark` and other tools (useful for
+contributors that don’t use VS Code).
+
+The configuration file is also used by [`remark-cli`][remark-cli], which is
+recommended to be used alongside `vscode-remark`, as an npm script and/or in
+CI, to enforce the markdown style.
+
+## Install
+
+[Get it on the VS Code Marketplace][marketplace] or install it by using Quick
+Open (<kbd>Ctrl</kbd> + <kbd>P</kbd>) and running the following:
+```txt
+ext install unifiedjs.vscode-remark
+```
+
+## Use
+
+To use this extension, set up [`remark-cli`][remark-cli] like you normally would
+in your project with a configuration file.
+The extension will find the configuration in your workspace just like running
+`remark-cli` in your terminal would.
+You will be able to see your linter work as you type and you can format your
+code if you want it to.
+
+Now, you can open markdown files in your project, and you’ll see squiggly lines
+and warnings in the `Problems` pane if the code doesn’t adhere to the coding
+standards.
+Here’s an example that should produce problems you can use to verify:
+
+```markdown
+1) Hello, _Jupiter_ and *Neptune*!
+```
+
+## Formatting
+
+This extension can format markdown files.
+
+To format a file, pull up the command pallete (<kbd>Ctrl</kbd> +
+<kbd>Shift</kbd> + <kbd>P</kbd>), choose `Format Document With…`, and select
+`remark`.
+
+To make `vscode-remark` the default formatter for markdown, add the following to
+your `settings.json` (which you can open with <kbd>Ctrl</kbd> + <kbd>,</kbd>):
+
+```json
+{
+  "[markdown]": {
+    "editor.defaultFormatter": "unifiedjs.vscode-remark",
+  }
+}
+```
+
+Now markdown documents can be formatted using <kbd>Ctrl</kbd> +
+<kbd>Shift</kbd> + <kbd>I</kbd>.
+
+You can optionally choose to automatically format when saving with
+`editor.formatOnSave`:
+
+```json
+{
+  "[markdown]": {
+    "editor.defaultFormatter": "unifiedjs.vscode-remark",
+    "editor.formatOnSave": true
+  }
+}
+```
+
+## Plugins
+
+The **remark** ecosystem has a variety of plugins available.
+Most notably you’ll want to check out [`remark-lint`][remark-lint].
+See this curated [list of plugins][list-of-plugins] for more remark plugins.
+
+## Syntax
+
+remark follows CommonMark, which standardizes the differences between markdown
+implementations, by default.
+Some syntax extensions are supported through plugins.
```

Comment:
I’m not too sure how much this contributes here, I’m open to removing it, but also okay to keep it.

Reply:
Do you mean the entire `Syntax` section? To me it doesn’t really make a difference. It’s documented in other places as well.

Maybe it’s best to remove it to keep it simple?

Reply:
yep, I mean the whole section. So sounds like we agree on removing it?

Reply:
Yep, removed.